endpoints/getting-started/app.yaml

```diff
@@ -1,8 +1,6 @@
 runtime: go
 env: flex
 
-beta_settings:
-  # Enable Google Cloud Endpoints API management.
-  use_endpoints_api_management: true
-  # Specify the Open API specification.
-  endpoints_swagger_spec_file: openapi.yaml
+endpoints_api_service:
+  name: ENDPOINTS SERVICE-NAME # To be replaced.
+  config_id: ENDPOINTS CONFIG-ID # To be replaced.
```

Comment:
To be replaced by what?

Should these be `ENDPOINTS-SERVICE-NAME` to be consistent with `YOUR-PROJECT-ID`?